src/controllers/client/client.router.ts

```diff
@@ -1,12 +1,13 @@
 import express from 'express';
+import ListController from './list.controller';
 import GetController from './get.controller';
-import PostInfoController from './post.info.controller';
 import PostController from './post.controller';
+import { assertAssetPoolAccess } from '@/middlewares';
 
 const router = express.Router();
 
-router.get('/', GetController.controller);
-router.post('/info', PostInfoController.controller);
-router.post('/', PostController.controller);
+router.get('/', assertAssetPoolAccess, ListController.controller);
```

Comment:
@GarfDev You did not add access middleware here, so everyone could potentially request those endpoints. 

You are also not validating JWT scopes (see examples in other routers). It was probably not mentioned in the dev tasks, but should something that is part of development for such a feature by default.